src/helpers/highlight.ts

```diff
@@ -15,14 +18,16 @@ export default function highlight({
   attribute,
   highlightedTagName = 'mark',
   hit,
+  cssClasses = {},
 }: HighlightOptions): string {
   const attributeValue =
     (getPropertyByPath(hit, `_highlightResult.${attribute}.value`) as string) ||
     '';
 
-  const className = suit({
-    descendantName: 'highlighted',
-  });
+  const className =
```

Comment:
We can use `cx` here:

```ts
import cx from 'classnames';

// ...

const className = cx(
  suit({
    descendantName: 'highlighted',
  }),
  cssClasses.highlighted
);
```

Reply:
cx is currently not a dependency to the connectors, using it here would make it a dependency for Vue & Angular

Reply:
Ah, makes sense! Might be worth explaining that we don't want to use the `classnames` dependency here because it's used in other flavors, then?

Another suggestion would be to use:

```ts
const className = [
  suit({
    descendantName: 'highlighted',
  }),
  cssClasses.highlighted,
]
  .filter(Boolean)
  .join(' ');
```

That's rather a matter of taste, so feel free to keep it that way.

Reply:
```suggestion
  // cx is not used, since it would be bundled as a dependency for Vue & Angular
  const className =
```